st2common/st2common/transport/bootstrap_utils.py

```diff
@@ -0,0 +1,68 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kombu import Connection
+from st2common import log as logging
+from st2common.transport import utils as transport_utils
+from st2common.transport.connection_retry_wrapper import ConnectionRetryWrapper
+from st2common.transport.execution import EXECUTION_XCHG
+from st2common.transport.liveaction import LIVEACTION_XCHG
+from st2common.transport.reactor import TRIGGER_CUD_XCHG, TRIGGER_INSTANCE_XCHG
+from st2common.transport.reactor import SENSOR_CUD_XCHG
+
+LOG = logging.getLogger('st2common.transport.bootstrap')
+
+__all__ = [
+    'register_exchanges'
+]
+
+EXCHANGES = [EXECUTION_XCHG, LIVEACTION_XCHG, TRIGGER_CUD_XCHG, TRIGGER_INSTANCE_XCHG,
+             SENSOR_CUD_XCHG]
+
+
+def _do_register_exchange(exchange, connection, channel, retry_wrapper):
```

Comment:
Well I suspect we will add more junk here later. I don't care either way what this is named. Can make it a class etc as well.